samples/snippets/quickstart_get_saved_query.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import argparse
+
+
+def get_saved_query(saved_query_name):
+    # [START asset_quickstart_get_saved_query]
+    from google.cloud import asset_v1
+
+    # TODO saved_query_name = 'SavedQuery Name you want to get'
+
+    client = asset_v1.AssetServiceClient()
+    response = client.get_saved_query(request={"name": saved_query_name})
+    print("gotten_saved_query: {}".format(response))
+    # [START asset_quickstart_get_saved_query]
```

Comment:
Snippet bot complaints:

```suggestion
    # [END asset_quickstart_get_saved_query]
```